iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/convert.inl

```diff
@@ -137,26 +138,26 @@ inline bool convert::stringIsNumberWithErrorMessage(const char* v, const NumberT
 {
     if (!stringIsNumber(v, type))
     {
-        std::cerr << v << " is not ";
+        IOX_LOG(WARN) << v << " is not ";
         switch (type)
         {
         case NumberType::FLOAT:
         {
-            std::cerr << "a float";
+            IOX_LOG(WARN) << "a float";
             break;
         }
         case NumberType::INTEGER:
         {
-            std::cerr << "a signed integer";
+            IOX_LOG(WARN) << "a signed integer";
             break;
         }
         case NumberType::UNSIGNED_INTEGER:
         {
-            std::cerr << "an unsigned integer";
+            IOX_LOG(WARN) << "an unsigned integer";
             break;
         }
         }
-        std::cerr << std::endl;
+        IOX_LOG(WARN);
```

Comment:
This can be removed since the logger automatically adds a new line at the end of a log message.